app/Services/UserService.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Services;
+
+use Exception;
+use App\Models\User;
+use App\Http\Requests\UserRequest;
+use Illuminate\Support\Facades\Auth;
+
+class UserService
+{
+    public function authenticateUser(UserRequest $request)
```

Comment:
@sabinshrestha8 Saying this again don't pass request object in your domain layer